### ISSUE TITLE
Device/entities do not seem to be loading properly. Refresh Token Issue?

### ISSUE THREAD

**Issue by ThomYoung** [kbirger/ha-smarter-integration#78]
## Version of the custom_component

v0.4.0-dev.0
## Configuration

This error originated from a custom integration.

Logger: custom_components.smarter
Source: custom_components/smarter/smarter_hub.py:53
integration: Smarter Kettle and Coffee (documentation, issues)
First occurred: 9:34:08 AM (1 occurrences)
Last logged: 9:34:08 AM

failed to log in with provided refresh token. 'SmarterClient' object has no attribute 'sign_in_with_refresh_token'
## Describe the bug

Firstly, I wanted to say thank you for your work on this! I have had an iKettle for probably about 6 years now. Using IFTTT to automate the iKettle is just awful, the service breaks all the time and I have never understood why Smater takes that approach. I was really excited when I found this, this morning. There seems to be an issue where, after installing, all the entities show as unavailable or permanently on. This includes the boiling switch entity. I can actually start the kettle boiling by turning the switch of and immediately on again.

I have added a screenshot of what I am seeing in the integration below. Please let me know if you need anything else.

## Debug log

```text

Add your logs here.

```
![Capture](https://github.com/user-attachments/assets/e1fff534-11c3-4ccf-8335-9297ec923c54)

**Comment by kbirger** [comment]
Yeah, the provided apis don't actually seem to support logging in with refresh token. It is not that important anyway since credentials need to be stored one way or another, and are sent over TLS.

Latest dev build removes this code, so I'm closing the issue. Please create a new one if you have more trouble.
